nx-X11/programs/Xserver/hw/nxagent/Image.c

```diff
@@ -98,9 +98,9 @@ int nxagentAlphaCompat  = 0;
  * displays having different byte order.
  */
 
-extern void BitOrderInvert(unsigned char *, int);
-extern void TwoByteSwap(unsigned char *, register int);
-extern void FourByteSwap(register unsigned char *, register int);
+extern void nxagentBitOrderInvert(unsigned char *, int);
```

Comment:
Indeed. Now added. (Luckily, I had not done a git clean in my working copy since I wrote down the changes on this branch).